### DISCUSSION TITLE
Using media manager on frontend - report

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#5219] [Q&A Core]
Hi.
maybe somebody will know what to do... 

I am using media manager upload on frontend  - form handler, renderElement() with type "images". I tried admin UI too.

1. it doesn't work in bootstrap5  

![image](https://github.com/e107inc/e107/assets/5429548/01012099-cca2-46af-b6d0-63fceff5ddf4)

2. switching to bootstrap3 looks:

![image](https://github.com/e107inc/e107/assets/5429548/2cab9b0e-dda7-4ad3-a920-8bd876e3b41c)

3. user ends with admin login screen

![image](https://github.com/e107inc/e107/assets/5429548/8c5c36e3-6b63-4d0c-9afa-d74c4938d0f5)

4. if an admin account is used
no clear button on frontend so you can't delete image

![image](https://github.com/e107inc/e107/assets/5429548/e069e1be-89bb-493a-a68e-e8ace93c81fb)

It really looks like there is no other option than to make members admins by default.

**Comment by Alex-e107nl** [comment]
Hi @Jimmi08 i have to see it in my browser to give an answer :-) You cannot make all members admin though hihihi

**Reply by Jimmi08** [reply]
You can do it in trigger with custom code. But it doesn't solve issue... you need to allow them upload images (next admin right) and you can't limit them to their own images in admin area.

**Comment by Jimmi08** [comment]
I gave up. Maybe in 5 years, somebody other will need this and it will be solved. I will use WordPress media manager to solve this.
